### ISSUE TITLE
Cropped Price in Payment Request on Chrome

### ISSUE THREAD

**Issue by leonardola** [Automattic/woocommerce-payments#3920]
### Describe the bug
When following [these](https://github.com/Automattic/woocommerce-payments/wiki/Testing-instructions-for-critical-flows#checkout-with-payment-request) instructions. The Modal that shows up ends up cutting important information

![paynow](https://user-images.githubusercontent.com/6342964/157288471-68054a09-12d1-4e7d-8b73-3894a4fc93d6.jpeg)

This seems related to the site name/charge descriptor. 

We'll need to investigate if it is possible to fix this as it happens in a browser modal and we probably do not have control over it.

### To Reproduce
Follow [these](https://github.com/Automattic/woocommerce-payments/wiki/Testing-instructions-for-critical-flows#checkout-with-payment-request) steps

### Actual behavior
Price should show up and not be cropped 

### Desktop:
* Browser chrome
* Windows 10

**Comment by waclawjacek** [comment]
Just to confirm, looks like this is on Windows 10, correct?

**Comment by leonardola** [comment]
Yep. Just confirmed with Global Step and they were using Windows 10 and Chrome

**Comment by timur27** [comment]
HI @leonardola, thanks for all the details! 
May I ask which Chrome version Global Step used when the issue occurred? 
In the meantime, I checked a few versions, and below 🔽 are my findings so far. 

## OS used: Windows 10 via [Browserstack](https://www.browserstack.com/). 
### **Google Chrome `ver.73.0.3683.86`**
![image](https://user-images.githubusercontent.com/22319444/185893045-17661620-7243-4c85-a3c1-e725cbd3bfda.png) 
_Image link: https://d.pr/i/wsOXoW_

As we can see, with Chrome ver. `73.x`, information in `Contact info, Shipping address` is flattened into a single row, and the Shipping method field does not appear. All this leads to the lack of a vertical scroll bar on the right side. At the same time, one might also notice that the margin is big enough for the price not to be cropped even with the scroll bar in place.

> Note: zooming in while in the browser did not change the font size and thus the Modal behavior. 

### **Google Chrome `ver. 104.0.5112` (one of the latest versions - from 2nd August 2022)**
![image](https://user-images.githubusercontent.com/22319444/185898898-8cc08d84-0dd4-4e31-aeaa-acc1655b564c.png)
_Image link: https://d.pr/i/Oy7F4y_

![image](https://user-images.githubusercontent.com/22319444/185898950-e73e7673-f05a-4889-b804-7ba7d2b53b8b.png)
_Image link: https://d.pr/i/DmdRWi_

With the latest version of Google Chrome, we can see how the modal dialog design has evolved. There is a scroll bar, but the price is shown in another place. Thus, the issue we discuss would not be valid within that version.

#### To sum this up - I think we need to answer two questions: 
1. Are we able to change the behavior of the Modal Window? Based on how the design is changing from version to version, I guess this is not something we have control over.
2. What Chrome version did Global Step use when the issue occurred?

**Comment by leonardola** [comment]
@timur27 as the place that the price is shown changed and we don't se the problem on newer version I think we can just close this as a won't fix. It's been a while since the test was executed so it's hard to tell the exact version. But if we consider they were up to date they were probably using 99.0.4844 according to [wikipedia](https://en.wikipedia.org/wiki/Google_Chrome_version_history). 

Thanks for investigating this.
